docs/src/13-UI-Kit/input/README.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Input
+---
+
+# Migrating the Input Component
+
+## Sources
+
+- [Github](https://github.com/PrestaShopCorp/puik/tree/main/packages/components/input)
+- [Storybook](https://uikit.prestashop.com/?path=/story/components-input--default)
+
+## PrestaShop UIKit
+
+Original [bootstrap component](https://getbootstrap.com/docs/4.0/components/forms/) used in the PrestaShop UIKit:
+
+```html
+<input class="form-control form-control-lg" type="text" placeholder="Placeholder">
+```
+
+## PUIK
+
+### Basic Use
+
+For any input component that you use, replace the structure above with the following structure:
+
+```html
+<div class="puik-input">
+  <div class="puik-input__wrapper">
+    <input id="id" class="puik-input__field" placeholder="Placeholder" type="text">
+  </div>
+</div>
+```
+
+### Parameters
+
+You can use the usual attributes that can be found on an input tag (`placeholder`, `disabled`, `type`, `min`, `max`, `step`, etc.).
```

Comment:
Good, maybe you could precise the same thing for checkbox component : "You can use the usual attributes that can be found on an input of type checkbox (`disabled`, `checked`...etc)